build/stf-run-ci/tasks/main.yml

```diff
@@ -13,11 +15,22 @@
     sg_bridge_image_path: "{{ __internal_registry_path }}/{{ namespace }}/sg-bridge:{{ sg_bridge_image_tag }}"
     prometheus_webhook_snmp_image_path: "{{ __internal_registry_path }}/{{ namespace }}/prometheus-webhook-snmp:{{ prometheus_webhook_snmp_image_tag }}"
 
+- name: Set default image paths for bundle and index builds
+  set_fact:
+    sgo_bundle_image_path: "{{ __internal_registry_path }}/{{ namespace }}/smart-gateway-operator-bundle:{{ sgo_bundle_image_tag }}"
+    sto_bundle_image_path: "{{ __internal_registry_path }}/{{ namespace }}/service-telemetry-operator-bundle:{{ sto_bundle_image_tag }}"
+    stf_index_image_path: "{{ __internal_registry_path }}/{{ namespace }}/service-telemetry-framework-index:{{ stf_index_image_tag }}"
+
 - name: Fail on mutually exclusive flags
   fail:
     msg: __deploy_from_bundles_enabled not currently supported with __local_build_enabled (but should be)
```

Comment:
I think this is going to set us up to do this... I also have an idea for a "fast mode" where we can just `oc import-image` from upstream quay.io instead of building artifacts every time. If we're using content from `master` it's basically available to us through release-automation repo.

Only build images when same-named branch logic is in use.

Reply:
As long as this is tested with __deploy_from_bundles_enabled like in downstream CI, you can defer the rest.

Reply:
Ah fair point. I haven't verified I didn't regress something here.

Reply:
Related PR based on some testing at #436

Reply:
Frankly at this point, I'm kind of tired of looking at this and moving things around. I'm going to open a separate issue about moving this out and getting local builds of bundles + bundle deployments to work, but I don't want to do it here and now. You've convinced me there is a use case for it, so I'll get to it, but I need to get this landed and move onto some other pressing documentation issues.

Reply:
Tracking this in #434